pages/docs/configuration/configuration-coupling-multi.md

```diff
@@ -7,7 +7,7 @@ summary: "If you want to couple more than two participants, there are two option
 
 ## Composition of bi-coupling schemes
 
-To combine multiple coupling schemes, simply add them one after the other in the configuration:
+To combine multiple coupling schemes, add them one after the other in the configuration:
```

Comment:
This simplicity is a selling feature of preCICE. Maybe we can write it in a way that still implies that. How about:

```suggestion
preCICE allows us to combine multiple coupling schemes by only specifying them one after the other in the configuration:
```

I also like the "boils down to" that you used somewhere else.